src/queries/match-phrase-prefix.js

```diff
@@ -0,0 +1,27 @@
+import _ from 'lodash'
+
+/**
+ * Construct a Match Phrase Prefix query.
+ *
+ * @param  {String} field Field name to query over.
+ * @param  {String} term  Query value.
+ * @param  {Object} opts  See match_phrase_prefix
+ * @return {Object}       Match match_phrase_prefix.
+ */
+export default function matchPhrasePrefix(field, term, opts = {}) {
```

Comment:
This looks good and would be happy to look at it in a separate PR to release it more quickly.